pydrad/parse/parse.py

```diff
@@ -83,9 +83,21 @@ def __getitem__(self, index):
                           master_time=self._master_time,
                           **self._profile_kwargs)
         else:
+            # NOTE: This is a shortcut that allows for quick indexing if you are not taking a slice
```

Comment:
Not sure I understand this comment.  This occurs when a scalar is passed into the function, so the profile at a single time step, but then it passes an index of `None`?

Reply:
I moved this comment lower down so it is more obvious to which bit of code it applies.